src/main/java/oss/fosslight/util/DateUtil.java

```diff
@@ -608,6 +608,10 @@ public static int getMinutes(String date1, String date2) {
 		Calendar cal1 = convertStringToCalender(date1);
 		Calendar cal2 = convertStringToCalender(date2);
 
+		if(cal1 == null || cal2 == null) {
+			return -1;

```

Comment:
@soimkim 
You should hear the maintainer's opinion on the return value.

Reply:
Looking at the calling this function, it is OK because it doesn't seem to cause an error even if it returns a negative number.